docs/index.html

```diff
@@ -1,13 +1,13 @@
 ---
 
-title: statsforecast
```

Comment:
Can you please restore this document and the gif? I just realized that the html in the index notebook messes with the docs online. I'll open a different issue to work on that.

Reply:
Ok. Will do that.